code/game/objects/items/devices/laserpointer.dm

```diff
@@ -142,11 +142,10 @@
 				crystal_stack.use_tool(src, user, amount = 1) //use only one if we were installing from a stack of crystals
 			return
 		//the single crystal that we actually install
-		var/obj/item/stack/ore/bluespace_crystal/single_crystal = crystal_stack.split_stack(null, 1)
+		var/obj/item/stack/ore/bluespace_crystal/single_crystal = crystal_stack.split_stack(1)
 		if(isnull(single_crystal))
 			return
-		if(!user.transferItemToLoc(single_crystal, src))
-			return
+		single_crystal.forceMove(src)
```

Comment:
what's goin on here?

Reply:
transferItemToLoc is used here because in the old split_stack behavior it inserts the newly created stack into the user (puts in hands), since new split_stack sticks the new stack in null space we can just be using forceMove

Reply:
ook